src/functions-reference/real-valued_basic_functions.Rmd

```diff
@@ -1001,8 +1001,10 @@ standard normal cumulative distribution function of x
 \index{{\tt \bfseries inv\_phi }!{\tt (T x): R}|hyperpage}
 
 `R` **`inv_Phi`**`(T x)`<br>\newline
-standard normal inverse cumulative distribution function of p,
-otherwise known as the quantile function
+Return the standard normal inverse cumulative distribution function of p,
```

Comment:
There's no variable p here.  The return is a value, not a function.  I think the first line of this could be rewritten as

```
Return the value of inverse standard normal cdf $\Phi^{-1}$ at the specified quantile `x`.
```

Then rather than "values between (L, U)", I'd say "quantile arguments in (L, U)".

And then rather than "Values smaller than 1e-16", I'd say "Quantile arguments below 1e-16 are untested; quantiles above 0.999999999 result in increasingly large errors". 

This is OK, but I wouldn't be surprised if accuracy was still good for arguments below 1e-16.